taichi/system/unified_allocator.cpp

```diff
@@ -63,6 +64,18 @@ UnifiedAllocator::UnifiedAllocator(std::size_t size, Arch arch, Device *device)
 #else
     TI_NOT_IMPLEMENTED
 #endif
+  }
+  // This is an intermediate state.
+  // We will use memory pools to implement `Device::allocate_memory` soon.
+  else if (arch_ == Arch::x64 && false) {
```

Comment:
qq: when do we want to get rid of the `&& false` here?

Reply:
Ah nice catch. They shouldn't be here. I was debugging something and forgot to delete them.